### ISSUE TITLE
make pdf fails to build

### ISSUE THREAD

**Issue by scopatz** [swcarpentry/collaborative-lesson-development#30]
Building the PDF seems to fail with the following error.

```bash
scopatz@artemis ~/collaborative-lesson-development master $ make pdf
pdflatex paper
This is pdfTeX, Version 3.14159265-2.6-1.40.17 (TeX Live 2016/Debian) (preloaded format=pdflatex)
 restricted \write18 enabled.
entering extended mode
(./paper.tex
LaTeX2e <2017/01/01> patch level 3
Babel <3.9r> and hyphenation patterns for 3 language(s) loaded.
(/usr/share/texlive/texmf-dist/tex/latex/base/article.cls
Document Class: article 2014/09/29 v1.4h Standard LaTeX document class
(/usr/share/texlive/texmf-dist/tex/latex/base/size10.clo))
...
Output written on paper.pdf (7 pages, 229939 bytes).
Transcript written on paper.log.
bibtex paper
This is BibTeX, Version 0.99d (TeX Live 2016/Debian)
The top-level auxiliary file: paper.aux
The style file: plos2015.bst
I found no \bibdata command---while reading file paper.aux
Warning--I didn't find a database entry for "hlw"
Warning--I didn't find a database entry for "wiggins-mctighe"
Warning--I didn't find a database entry for "lessons-learned"
Warning--I didn't find a database entry for "instructor-training"
Warning--I didn't find a database entry for "how-to-teach-programming"
Warning--I didn't find a database entry for "git-survey"
Warning--I didn't find a database entry for "shell2015"
Warning--I didn't find a database entry for "shell2017"
Warning--I didn't find a database entry for "choral-explanations"
Warning--I didn't find a database entry for "producing-oss"
(There was 1 error message)
Makefile:13: recipe for target 'pdf' failed
make: *** [pdf] Error 2
```

**Comment by gvwilson** [comment]
Fixed in 164b2a3

**Comment by scopatz** [comment]
Thanks!
